pkg/controller/config.go

```diff
@@ -243,6 +245,8 @@ func InitFlags(config *ControllerConfig) {
 	flag.StringVar(&config.LBType, "loadbalancer", lbTypeAci, "Loadbalancer")
 	flag.IntVar(&config.MaxCSRTunnels, "max-csr-tunnels", 16, "Number of CSR tunnels")
 	flag.IntVar(&config.CSRTunnelIDBase, "csr-tunnel-id-base", 4001, "CSR starting tunnel ID")
+	flag.BoolVar(&config.RestartAfterLiveMigration, "restart-after-live-migration,omitempty", true,
```

Comment:
Not sure if we need the flag here, @snaiksat ?

Reply:
per offline conversation, we are good on this?

Reply:
Yes